FinniversKit/Sources/Fullscreen/FavoriteAdsList/Subviews/FavoriteAdsListTableHeader.swift

```diff
@@ -53,16 +53,39 @@ class FavoriteAdsListTableHeader: UIView {
         return CGRect(origin: contentStackView.frame.origin, size: titleLabel.frame.size)
     }
 
+    // MARK: - Tori NMP onboarding (Temporary)
+
+    var viewModel: FavoriteAdsListViewModel? {
+        didSet {
+            panelText = viewModel?.panelTextOnboarding ?? ""
+            showPanel = viewModel?.showPanelForOnboarding ?? false
+        }
+    }
+
+    var showPanel: Bool = false {
+        didSet {
+            infoPanel.isHidden = !showPanel
+        }
+    }
+
+    var panelText: String = "" {
+        didSet {
+            let model = PanelViewModel(text: panelText)
+            infoPanel.configure(with: model)
+        }
+    }
+
     // MARK: - Private properties
 
     private lazy var tapRecognizer = UITapGestureRecognizer(target: self, action: #selector(handleViewTap))
 
     private lazy var contentStackView: UIStackView = {
-        let stackView = UIStackView(arrangedSubviews: [titleLabel, subtitleView, searchBar, sortingContainerView])
+        let stackView = UIStackView(arrangedSubviews: [titleLabel, subtitleView, infoPanel, searchBar, sortingContainerView])
```

Comment:
If there is no onboarding, shouldn't it be better to not have the infoPanel at all? it seems we are putting an empty view in case of no onboarding.

Reply:
I might be confused here, but since we are getting the `panelTextOnboarding` regardless of which target we are running, we need `showPanelForOnboarding` also to avoid displaying for `finn`? So even though we don't have a case where we want to have text without the panel it self, we do have the text (since it it generated for all the targets simultaneously). And therefore need to know whether or not to show it🤔